design-documents/graph-ql/coverage/customer/gift-registry.graphqls

```diff
@@ -128,40 +130,68 @@ input GiftRegistryDynamicAttributeInput {
     value: String!
 }
 
-type CreateGiftRegistryOutput {
-    gift_registry: GiftRegistry
+interface GiftRegistryUserErrors {
+    status: Boolean!
+    gift_registry_user_errors: [GiftRegistryItemsUserError]!
```

Comment:
@eduard13 check this out https://github.com/magento/architecture/pull/480#discussion_r618590876